.github/workflows/torch_setup.yml

```diff
@@ -133,4 +133,4 @@ jobs:
     - name: PyTest
       if: ${{ (success() || failure()) && (steps.install.outcome == 'failure' || steps.install.outcome == 'success') }}
       shell: bash -l {0}
-      run: pytest -v -m torch deepchem
+      run: pytest -v -m 'torch and not slow' deepchem
```

Comment:
Let's keep running the slow tests. It seems to be working out so far and provides us with a more robust CI